speed read test does not compile on hdf5 1.8

there is a stupid mistake which causes for hdf5 1.8
```
                                ^
[ 73%] Building CXX object test/CMakeFiles/unit_tests.dir/node/dataset_read_speed_test.cpp.o
/home/jkotan/sources/h5cpp/test/node/dataset_read_speed_test.cpp: In member function ‘virtual void DatasetReadSpeedTest::SetUp()’:
/home/jkotan/sources/h5cpp/test/node/dataset_read_speed_test.cpp:63:5: error: ‘dcpl’ was not declared in this scope
     dcpl.layout(property::DatasetLayout::CHUNKED);
     ^
/home/jkotan/sources/h5cpp/test/node/dataset_read_speed_test.cpp:68:18: error: ‘lcpl’ was not declared in this scope
            space,lcpl,dcpl,dapl);
                  ^
/home/jkotan/sources/h5cpp/test/node/dataset_read_speed_test.cpp:68:28: error: ‘dapl’ was not declared in this scope
            space,lcpl,dcpl,dapl);
```
I will create a fix.

--- a/test/node/dataset_read_speed_test.cpp
+++ b/test/node/dataset_read_speed_test.cpp
@@ -43,15 +43,15 @@ class DatasetReadSpeedTest : public testing::Test
 #if H5_VERSION_GE(1, 10, 0)
     property::FileCreationList fcpl;
     property::FileAccessList fapl;
-    property::DatasetCreationList dcpl;
-    property::LinkCreationList lcpl;
-    property::DatasetAccessList dapl;
     fapl.library_version_bounds(property::LibVersion::LATEST,
                                 property::LibVersion::LATEST);
     file::File f = file::create("dataset_read_speed.h5", file::AccessFlags::TRUNCATE, fcpl, fapl);
 #else
     file::File f = file::create("dataset_read_speed.h5", file::AccessFlags::TRUNCATE);
 #endif
+    property::DatasetCreationList dcpl;
+    property::LinkCreationList lcpl;
+    property::DatasetAccessList dapl;
     node::Group root = f.root();
     long long unsigned int xdim = 867;
     long long unsigned int ydim = 700;

